scripts/ci/linux/build.sh

```diff
@@ -13,11 +13,18 @@ echo "PLATFORM_CFLAGS += $CUSTOMFLAGS" >> libs/openFrameworksCompiled/project/li
 cd libs/openFrameworksCompiled/project
 make Debug
 
-echo "Building emptyExample"
```

Comment:
@arturoc why did you delete this part? I don't think a CI build is really sensible if we can't even build the emptyexample. Even if we don't want to build that temporarily, why delete this and not simply comment it out and adding some explanation?